R/dictionaries.R

```diff
@@ -153,7 +164,7 @@ setMethod("as.list",
 #'   the Linguistic Inquiry and Word Count software} \item{\code{"yoshikoder"}}{
 #'   format used by Yoshikoder software} \item{\code{"lexicoder"}}{format used
 #'   by Lexicoder} \item{\code{"YAML"}}{the standard YAML format}}
-#' @param concatenator the character in between multi-word dictionary values. 
+#' @param separator the character in between multi-word dictionary values. 
 #'   This defaults to \code{"_"} except LIWC-formatted files, which defaults to 
```

Comment:
I think we should default it to `" "`. What dictionary format do we know uses `_` for the separator?